cloud/services/container/nodepools/reconcile.go

```diff
@@ -317,6 +330,15 @@ func (s *Service) updateNodePoolSize(ctx context.Context, setNodePoolSizeRequest
 	return nil
 }
 
+func (s *Service) updateNodePool(ctx context.Context, updateNodePoolRequest *containerpb.UpdateNodePoolRequest) error {
+	_, err := s.scope.ManagedMachinePoolClient().UpdateNodePool(ctx, updateNodePoolRequest)
+	if err != nil {
+		return err
```

Comment:
It would be good to warp this error to give some extra context.

Reply:
Updated `updateNodePool` caller function. Hope it is fine now.